internal/provider/scorecard_resource_test.go

```diff
@@ -53,7 +53,6 @@ resource %[1]q %[2]q {
     ]
   }
   filter = {
-    category = "SERVICE"
```

Comment:
Thanks for removing this.   I can file an internal bug to get the provider updated to support include/exclude types (which is what replaced category).